### PR TITLE
Lazy load top 200 cryptos

### PR DESCRIPTION
**Pull request by lampvux**
add infinite load top 200 cryptos symbols

### CODE REVIEW

File: frontend/modules/cryptologocheck.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    try {
+        
+        const topcoins = await axios.get(`https://api.finage.co.uk/list/cryptocurrency?apikey=${process.env.FINAGE_API_KEY}&limit=200`);
+        return await topcoins.data.results;        
+    } catch (error) {
+        console.log(error)
+    }
+
+}
+async function getSingleCoin(symbol){
```
Comment (pixelomo) [inline frontend/modules/cryptologocheck.js:16]: this looks good however I'm not seeing missing logos being fetched or the cryptologo.scss being created when running locally

**Review by pixelomo — CHANGES_REQUESTED** [review]
The infinite loader works nicely however I notice that the prices are being fetched for all coins. For example DAI previously showed price.

The logo fetch logic looks good but I don't see it working, fetching logos and generating a cryptologos.scss

Many of the new coins (51-200) have errors when viewing their item page, finage data fetches fail, the price is $0 or the page doesn't load at all for example /cryptocurrency/huobi-btc
_(This can be split into a new ticket we can resolve after this branch is merged)_

File: frontend/components/IndexList.vue [hunk 2]
```diff
@@ -64,7 +64,7 @@
       :to="`/${type}/${index.name.replace(/\s+|[' '\/]/g, '-').toLowerCase()}`"
     >
       <div class="d-flex icon-wrapper" :class="index.change > 8 && type === 'cryptocurrency' ? 'fire' : ''">
-        <div class="icon" :class="index.icon ? index.icon : index.symbol.toLowerCase()" :id="index.symbol.toUpperCase()"   />
+        <div class="icon" :class="index.icon ? index.icon : 's-' + index.symbol.toLowerCase()" :id="index.symbol.toUpperCase()"   />
```
Comment (lampvux) [inline frontend/components/IndexList.vue:67]: fix icon logo not load in homepage

File: frontend/pages/cryptocurrency/index.vue [hunk 3]
```diff
@@ -212,6 +212,10 @@ export default {
           if (response.results.length) {
             this.page++;
             this.cryptocurrency = this.cryptocurrency.concat(response.results);
+            // fetch crypto pricing on load
```
Comment (lampvux) [inline frontend/pages/cryptocurrency/index.vue:215]: fetch crypto price on load

**Pull request merged**